src/Routers/UsersRouter.js

```diff
@@ -139,11 +139,136 @@ export class UsersRouter extends ClassesRouter {
     });
   }
 
+  handleCreate(req) {
+    return rest
+      .create(
+        req.config,
+        req.auth,
+        this.className(req),
+        req.body,
+        req.info.clientSDK,
+        req.info.context
+      )
+      .then(response => {
+        response.response.oauth = req.config.oauth === true ? true : false;
+        if (req.config.oauth === true) {
+          const token = Auth.createJWT(
+            response.response.sessionToken,
+            req.config.oauthKey,
+            req.config.oauthTTL
+          );
+          response.response.accessToken = token.accessToken;
+          response.response.expires_in = token.expires_in;
+          delete response.response.sessionToken;
+        }
+        return response;
+      });
+  }
+
+  handleRefresh(req) {
+    const payload = req.body;
+    const { client, code } = payload;
+
+    if (!client || !code) {
+      throw new Parse.Error(Parse.Error.INVALID_SESSION_TOKEN, 'Invalid session token');
+    }
+
+    // Consulta
```

Comment:
Can you write the comments in English or remove them if they are unnecessary?

Reply:
@dplewis, I'm having trouble setting up the tests. Through the commit analyzer, I need to create the tests for the new functions.

Reply:
@dplewis I need help with test cases.